png_cicp_chunk_proposal.md

```diff
@@ -13,31 +13,33 @@ An existing W3C group note, [BT2100-in-PNG]  specifies an approach which is limi
 * extensible signaling of color space based on H.273
 * does not require the presence of iCCP chunk and embedded ICC profiles
 
+## Non Requirements
+* compatibility with HDR colorspace definitions in any other raster image format (eg. JPEG, AVIF)
```

Comment:
I am not sure why this non-requirement is necessary? Can you provide an example?

Reply:
It's necessary because it explains why we aren't "just using ICC profiles" (which would be the more compatible method).

Reply:
@lrosenthol I do not think it is a fair assessment that ICC profiles are the only way to do HDR in other raster formats. For example, they are not used in common post-production formats like MXF. Even recent versions of J2K supports both ICC profiles and H.273 signaling. I therefore do not agree that compatibility with other formats is a non-requirement.

Reply:
I agree with @palemieux that ICC profiles are not the only way.

The specific ones cited are particularly unfortunate; JPEG is a compression scheme, not an image format and AVIF is an image format, which mainly uses H.273 signalling (ICC profiles are also supported, but not widely used, as AVIF is basically a single-frame or few-frames video).

Reply:
I accept that ICC profiles wouldn't necessarily be the only consistent way across formats...but right now they are.  And my point is that we aren't concerned with consistency in this proposal

Reply:
So I'll change compatibility to consistency

Reply:
I think we should not have this callout.

As I explain below, we should allow both cICP and iCCP chunks. Compatibility & consistency remains if we allow both.
But even if we didn't, using a standard like H.273 is quite explicitly a form of compatibility & consistency. It's just a different form.

More generally—although the exact wording here is "other raster image format"—we should also consider compatibility & consistency with the web as a whole:
You can use a custom ICC profile in CSS, via the @color-profile at-rule [1].
But you can also use a predefined and enumerated color space, like color(rec2020 ...) [2]. This is very much like H.273.

We don't need every rec2020 image on a page to redefine what rec2020 means.

[1] https://www.w3.org/TR/css-color-4/#at-profile
[2] https://www.w3.org/TR/css-color-4/#predefined

Reply:
H.273 is not giving us consistency in rendering of video frames.
CICP is a color space, not a color rendering, so it supports a large number of renderings.
And that's how CICP is used today in video.
- One rendering for scene
- A different rendering for a reference display
- And multiple renderings for displays with non-reference peak nits.

CICP is the best we have today.

Reply:
Per 2021-02-24: we are not concerned with bit-identical serialization of the H.273 parameters